modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path

```diff
@@ -605,8 +605,8 @@
 	<td></td>
 </tr>
 <tr>
-	<td>ICON_TRASH_ROLE_TYPE</td>
-	<td>//div[@class='panel-body']//input[contains(@id,'notificationName') and contains(@value,'')]/../..//*[contains(@class, 'icon-trash')]</td>
```

Comment:
This doesn't work if it has more than one trash can icon. 

As a solution, I implement a index on the path.